mscore/plugin/api/qmlpluginapi.h

```diff
@@ -83,6 +83,11 @@ class PluginAPI : public Ms::QmlPlugin {
       /** current score, if any (read only) */
       Q_PROPERTY(Ms::PluginAPI::Score* curScore     READ curScore)
 //TODO-ws      Q_PROPERTY(QQmlListProperty<Ms::Score> scores READ scores)
+      /**
```

Comment:
this here will probably cause a merge conflict with #5021, but should be easy to fix